sdk/iot/Azure.Iot.Hub.Service/src/JobsClient.cs

```diff
@@ -4,22 +4,205 @@
+        /// </summary>
+        /// <param name="cancellationToken">Task cancellation token</param>
+        /// <returns>IEnumerable of JobProperties of all jobs for this IoT Hub.</returns>
+        public virtual Response<IReadOnlyList<JobProperties>> GetImportExportJobs(CancellationToken cancellationToken = default)
+        {
+            return _jobRestClient.GetImportExportJobs(cancellationToken);
+        }
+
+        /// <summary>
+        /// List all import and export jobs for the IoT Hub.
+        /// </summary>
+        /// <param name="cancellationToken">Task cancellation token</param>
+        /// <returns>IEnumerable of JobProperties of all jobs for this IoT Hub.</returns>
+        public virtual Task<Response<IReadOnlyList<JobProperties>>> GetImportExportJobsAsync(CancellationToken cancellationToken = default)
+        {
+            return _jobRestClient.GetImportExportJobsAsync(cancellationToken);
+        }
+
+        /// <summary>
+        /// Gets the import or export job with the specified ID.
+        /// </summary>
+        /// <param name="jobId">Id of the Job object to retrieve</param>
+        /// <param name="cancellationToken">Task cancellation token</param>
+        /// <returns>JobProperties of the job specified by the provided jobId.</returns>
+        public virtual Response<JobProperties> GetImportExportJob(string jobId, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(jobId, nameof(jobId));
+
+            return _jobRestClient.GetImportExportJob(jobId, cancellationToken);
+        }
+
+        /// <summary>
+        /// Gets the import or export job with the specified ID.
+        /// </summary>
+        /// <param name="jobId">Id of the Job object to retrieve</param>
+        /// <param name="cancellationToken">Task cancellation token</param>
+        /// <returns>JobProperties of the job specified by the provided jobId.</returns>
+        public virtual Task<Response<JobProperties>> GetImportExportJobAsync(string jobId, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(jobId, nameof(jobId));
+
+            return _jobRestClient.GetImportExportJobAsync(jobId, cancellationToken);
+        }
+
         /// <summary>
-        /// place holder
+        /// Cancels/Deletes the job with the specified ID.
         /// </summary>
-#pragma warning disable AZC0007 // DO provide a minimal constructor that takes only the parameters required to connect to the service.
-        public JobsClient()
-#pragma warning restore AZC0007 // DO provide a minimal constructor that takes only the parameters required to connect to the service.
+        /// <param name="jobId">Id of the job to cancel</param>
+        /// <param name="cancellationToken">Task cancellation token</param>
+        /// <returns>A response string object indicating result of the cancellation.</returns>
```

Comment:
Is this a json string that is returned? Is there any advice we can give to users here on how to parse it?